cupy/cuda/cupy_cusparse.h

```diff
@@ -0,0 +1,6 @@
+#ifndef INCLUDE_GUARD_CUPY_CUSPARSE_H
+#define INCLUDE_GUARD_CUPY_CUSPARSE_H
+
+#include <cusparse.h>
+
+#endif  // INCLUDE_GUARD_CUPY_CUSPARSE_H
```

Comment:
This file should be merged with `cupy_cuda.h`. Why there is `cupy_cudnn.h` that cuDNN is not distributed in CUDA itself.